cmd/mattermost/commands/command.go

```diff
@@ -67,3 +78,38 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func deleteCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Enter the commandID to delete")
+	}
+
+	confirmFlag, _ := command.Flags().GetBool("confirm")
+	if !confirmFlag {
+		var confirm string
+		CommandPrettyPrintln("Are you sure you want to delete the command specified?  All data will be permanently deleted? (YES/NO): ")
+		fmt.Scanln(&confirm)
+		if confirm != "YES" {
+			return errors.New("ABORTED: You did not answer YES exactly, in all capitals.")
+		}
+	}
+
+	commandID := args[0]
+	err = deleteCommand(a, commandID)
+	if err != nil {
+		CommandPrintErrorln("Unable to delete command '" + commandID + "' error: " + err.Error())
+		return err
+	}
+	CommandPrettyPrintln("Deleted command '" + commandID + "'")
+	return nil
+}
+
+func deleteCommand(a *app.App, commandID string) *model.AppError {
```

Comment:
I think we can remove this "wrapper function" because is unnecessary.

Reply:
Done.